src/test/resources/data/StudentProfilePageUiTest.json

```diff
@@ -37,13 +37,43 @@
       "isInstructor": false,
       "email": "SHomeUiT.benny.c@gmail.tmt",
       "institute": "TEAMMATES Test Institute 1"
+    },
+    "instructor1OfTestingSanitizationCourse": {
```

Comment:
Accounts are owned by students or instructors. Putting "instructor" in the name makes it easier to differentiate between accounts of students and accounts of instructors. It is quite common for accounts and the corresponding student/instructor to share the same key too, such as in typicalDatBundle.json, which I dont think is an issue. This can make finding the corresponsing instructor/student of an account much easier, especially when they are just an instructor/a student for a single course. What do you think?